packages/aws-cdk-lib/aws-rds/README.md

```diff
@@ -28,6 +28,24 @@ const cluster = new rds.DatabaseCluster(this, 'Database', {
 });
 ```
 
+To setup a Mutli AZ database cluster, define a `MutliAZDatabaseCluster`.
+
+```ts
+declare const vpc: ec2.Vpc;
+const cluster = new rds.MultiAZDatabaseCluster(this, 'Database', {
+  engine: rds.DatabaseClusterEngine.mysql({ version: rds.MysqlEngineVersion.VER_8_0_32 }),
+  credentials: rds.Credentials.fromGeneratedSecret('clusteradmin'), // Optional - will default to 'clusteradmin' username and generated password
+  instanceProps: {
```

Comment:
In a recent [pull request](https://github.com/aws/aws-cdk/pull/25437) I added a new API for adding instances to a cluster. Even though it was to enable adding serverless instances, I still think it is a better API that allows you to individually control the instances in the cluster. Can you modify the `MultiAZDatabaseCluster` to match? In this case we only need the new API.

Reply:
As far as I understand with Multi-AZ Database Clusters you don't specify instances, just the cluster and RDS handles creating the instances. If I am wrong about this please point me to the documentation where I can better understand.

Reply:
ah ok in that case you don't need the `instanceProps` because that is used to create the instances in the aurora cluster.